BUILD

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Google LLC
```

Comment:
Now that we have BUILD files, we should also add a presubmit check to ensure the build never breaks.
You can take [this one](https://github.com/p4lang/p4runtime/blob/main/.github/workflows/ci-build-proto.yml) as a template and just adjust the last two steps.

Reply:
Done